feat: Add option to lock to landscape when entering fullscreen

sbbullet opened the pull request:
solves #28

```js
    player.mobileUi({
      fullscreen: {
        enterOnRotate: true,
        exitOnRotate: true,
        lockOnRotate: true,
        alwaysLockToLandscape: true,
        iOS: false,
        disabled: false,
      },
      touchControls: {
        seekSeconds: 10,
        tapTimeout: 300,
        disableOnEnd: false,
        disabled: false,
      },
    });

    player.on('play', () => {
      if (!player.isFullscreen()) {
        player.requestFullscreen()
      }
    })

```

and the fullscreen mode switches to landscape when the user taps the fullscreen icon. Since a lot of video players in the market including YouTube video player switches to landscape mode and locks in it when the user taps a fullscreen button, this feature will mimick the youtube experience.

Also, README and index.html are updated with new options including the `disabled` option which I found that you haven't mentioned in the docs.

- [x] `npm run lint` passed
- [x] `npm run test` passed with all 12 tests as SUCCESS

mister-ben commented on README.md:
```diff
@@ -12,6 +12,7 @@ Fullscreen control:
 
 - Rotate to landscape to enter Fullscreen
 - Lock to fullscreen on rotate
+- Always lock fullscreen to landscape (works even when device rotation is disabled/non-functional)
```
```suggestion
- Always lock to landscape when entering fullscreen (works even when device rotation is disabled/non-functional)
```

mister-ben commented on src/plugin.js:
```diff
@@ -67,7 +68,7 @@ const onPlayerReady = (player, options) => {
     videojs.log.warn('videojs-mobile-ui: `fullscreen.iOS` is deprecated. Use Video.js option `preferFullWindow` instead.');
     if (videojs.browser.IS_IOS && videojs.browser.IOS_VERSION > 9 &&
-        !player.el_.ownerDocument.querySelector('.bc-iframe')) {
+      !player.el_.ownerDocument.querySelector('.bc-iframe')) {
```
```suggestion
        !player.el_.ownerDocument.querySelector('.bc-iframe')) {
```
I'd prefer to leave the multiline conditional spacing as it was

mister-ben commented on src/plugin.js:
```diff
@@ -112,8 +113,8 @@ const onPlayerReady = (player, options) => {
-        if (options.fullscreen.lockOnRotate &&
-            screen.orientation && screen.orientation.lock) {
+        if ((options.fullscreen.lockOnRotate || options.fullscreen.alwaysLockToLandscape) &&
+          screen.orientation && screen.orientation.lock) {
```
```suggestion
            screen.orientation && screen.orientation.lock) {
```

mister-ben reviewed (changes requested):
Thanks! The option should perhaps be `lockToLandscapeOnEnter` similar to the others, as it's clearer what action triggers the lock.

sbbullet commented:
Hello @mister-ben, thanks for your suggestions. I was in dilemma about how we might bring the functionality through the package and your suggestions made the path clearer on how we can implement it. I have updated the master branch with the suggested changes and updated the option name and README file as well. Thank you!

sbbullet commented:
Hello @mister-ben, I'm extremely sorry to say but I forgot to mention that [in this line](https://github.com/mister-ben/videojs-mobile-ui/blob/26c58d4521cfdf65f944e861e9b9d07bc384b5b8/src/plugin.js#L116), should we change the line `(options.fullscreen.lockOnRotate || options.fullscreen.lockToLandscapeOnEnter)` to just `options.fullscreen.lockOnRotate` since we can specify to not lock on rotate by setting `options.fullscreen.lockOnRotate=false` but if the player still stays in portrait mode when the player goes fullscreen then we forcefully lock to landscape by setting `options.fullscreen.lockToLandscapeOnEnter=true`. This way if the device has functional rotation functionality then we can go fullscreen only on rotate and when the device has no non-functional rotation then it will force the player to be in landscape mode. I'm extremely sorry to mention this dilemma I had & wanted to know what you think of it.
